libvcx/src/api_lib/api_handle/credential_def.rs

```diff
@@ -126,11 +141,13 @@ pub fn check_is_published(handle: u32) -> VcxResult<bool> {
     })
 }
 
-pub fn rotate_rev_reg_def(handle: u32, revocation_details: &str) -> VcxResult<String> {
+pub fn rotate_rev_reg_def(handle: u32, revocation_details: &str, new_tails_url: Option<String>) -> VcxResult<String> {
```

Comment:
When this is called I think we should return error if 
- `self.rev_reg` is None
And make `new_tails_url` parameter required, not optional

Reply:
We do return error if `self.rev_reg` is None